apps/codeforafrica/src/components/StayInTouch/StayInTouch.js

```diff
@@ -0,0 +1,91 @@
+import { Link } from "@commons-ui/next";
+import { Grid } from "@mui/material";
+import { styled } from "@mui/material/styles";
+import PropTypes from "prop-types";
+import * as React from "react";
+
+import Title from "./Title";
+
+const IconRoot = styled("img", {
+  slot: "Root",
+  name: "StayInTouchIcon",
+})(() => ({
+  width: "1.375rem",
+  height: "1.375rem",
+  objectFit: "contain",
+}));
+
+const StayInTouch = React.forwardRef(function StayInTouch(
+  { children, socialMedia, title, ...props },
+  ref
+) {
+  if (!socialMedia?.length) {
+    return null;
+  }
+
+  return (
+    <Grid
+      sx={{
+        marginTop: "3.125rem",
+        justifyContent: { xs: "center", md: "flex-start" },
+      }}
+      container
+      ref={ref}
+      {...props}
+    >
+      {title && (
+        <Grid
+          item
+          xs={12}
+          md={4}
+          sx={{ textAlign: { xs: "center", md: "left" } }}
+        >
+          <Title title="Stay in touch:" />
+        </Grid>
+      )}
+      <Grid
+        sx={{
+          width: { md: "auto" },
+          justifyContent: { md: "flex-start", xs: "center" },
+        }}
```

Comment:
We don't need `sx` I think... Grid props are responsive I think i.e. `<Grid justifyContent={{ xs: "center", md: "flex-start"` should work, no?